.github/workflows/test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
             hf_models: "sentence-transformers/all-roberta-large-v1"
           - name: "continuous batching"
             markers: "cpu and cb"
-            flags: "--timeout=300"
+            flags: "--timeout=300  --durations=0 -s"
```

Comment:
The CB tests are our most problematic in terms of runtime, so the `--durations` and `-s` flags are useful to debug what the tests are doing and how much time they're taking